.azure/pipelines/ci.yml

```diff
@@ -596,9 +596,9 @@ stages:
       agentOs: Windows
       timeoutInMinutes: 180
       steps:
-      - script: .\restore.cmd -ci
-        displayName: Restore
-      - script: .\build.cmd -ci -NoRestore -test -projects eng\helix\helix.proj /p:IsHelixJob=true /p:IsHelixDaily=true /p:BuildAllProjects=true /p:BuildNative=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
+      - script: ./build.cmd -ci -all -pack
+        displayName: Build shared fx
```

Comment:
Nice to have: If you only need one shared framework and it's the 64-bit one, suggest using 3 jobs instead of two: Build the x64 shared runtime, upload pipeline artifacts you need, then depend on that build in the two Helix jobs. Might also be able to use the x64 shared runtime job as a dependency for the Windows x64 / x86 job. (Question: Will we run any tests on x86 Helix agents or using the x86 Helix shared framework?)

Otherwise, comments above apply here too.

Reply:
Is there a reason we aren't doing that today with the templates tests building their own shared framework?

Reply:
Not really. But, we're getting to the point where building the same Windows bits all over the place is just wasting time. Storage is the same because the packages will become artifacts in any case and the network bandwidth to download things isn't that high. My only real question is whether it's worth uploading the artifacts/bin/ folder as a new Windows-only pipeline artifact for use in the Windows Test job.

One slight complication: We don't create package artifacts for public builds from source. Use pipeline artifacts in that case.

As I said, this is nice to have. We can deal w/ the 5 or so Windows x64 builds separately if you prefer.